server/config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     scriptCache: '24h',
     matchingRecordsToBeShownPerPage: 5,
     maximumPages: 4,
-    minimumMatchProbability: 0.5,
+    minimumMatchProbability: 0.0,
```

Comment:
Thanks @markreesmoj , this change achieves what is expected in the ticket besides catering to the possibility (which i see is there) that if at all we want to resort back to filtration, we can always change this. hence i think keeping this is a good idea

Reply:
This could change in the future

Reply:
Fair enough, note that code is a liability not an asset :-). 
I would argue that in the future this can be added back in if required. 
There are quite a few dead branches in the PiC codebase already.